scripts/assemble_changelog.py

```diff
@@ -108,6 +111,12 @@ def format_category(cls, title, body):
 class TextChangelogFormat(ChangelogFormat):
     """The traditional Mbed TLS changelog format."""
 
+    _unreleased_version_text = b'= mbed TLS x.x.x branch released xxxx-xx-xx'
```

Comment:
Minor, pre-existing: the word "branch" here bugged me ever since we started adding it (in Mbed TLS 2.3.0). I don't see in what sense every single patch release would be a "branch" and I don't see what useful information that word would convey in version title line.

Reply:
I agree. I'd also like to tweak the wording of category names: change “Bugfix” to “Bug fixes” (like “API changes” is not “APIchange”), replace “Changes” by more precise categories... But in this PR, the format does not change at all, not even section headers.

Reply:
Sure, no problem.